packages/manager/modules/billing/src/payment/fidelity/creditOrder/billing-fidelity-creditOrder.controller.js

```diff
@@ -76,11 +84,9 @@ export default /* @ngInject */ (
 
   $scope.displayBC = function displayBC() {
     $scope.resetAction();
-    $scope.setMessage(
-      $translate.instant('fidelity_creditOrder_step2_success', {
-        t0: $scope.creditOrder.BC.url,
-        t1: $scope.creditOrder.BC.orderId,
-      }),
+    Alerter.success(
+      $translate.instant('fidelity_creditOrder_submit_success'),
+      'billingPaymentFidelityAlert',
```

Comment:
does fidelity_creditOrder_submit_success will work without t0 here ?

Reply:
If you look at fidelity_creditOrder_submit_error, it is a complete string without parameter. The t0 is used when you have a parameter inside your string.

Reply:
Seen with @karim-ovh, this line will be fixed in next commit